site/en/docs/webstore/register/index.md

```diff
@@ -31,6 +31,6 @@ screen appears as shown here:
 Once you pay the registration fee and agree to the terms, you will not see this registration page
 again.
 
-<div class="aside aside--note"><strong>Group publishers</strong> note that only members of the group who've paid the registration fee can publish updates.</div>
+<div class="aside aside--note"><a href="https://developer.chrome.com/docs/webstore/group-publishers/"><strong>Group publishers</strong></a> note that only members of the group who've paid the registration fee can publish updates.</div>
```

Comment:
The target for this should be a path from the base:

<a href="/docs/webstore/group-publishers/">

Reply:
Other than that, looks great!